src/main/java/seedu/duke/project/Project.java

```diff
@@ -53,7 +63,7 @@ public void deleteTask(int index) {
     @Override
     public String toString() {
         
-        return description;
+        return projectName;
```

Comment:
I feel to make it more convenient you can return name, description and deadline here.